packages/rum-recorder/src/index.ts

```diff
@@ -13,3 +13,10 @@ export {
   RumResourceEvent,
   RumViewEvent,
 } from '@datadog/browser-rum-core'
+
+export {
+  Segment as internal_Segment,
+  CreationReason as internal_CreationReason,
+  IncrementalSource as internal_IncrementalSource,
+  RecordType as internal_RecordType,
+} from './types'
```

Comment:
what about not prefixing by internal?

Reply:
I don't want to expose those values publicly, that's why I added the prefixes. We could just not document them, but editors autocompletion would still suggest them when importing the recorder.

Another simple solution would be to use a direct path in e2e tests, instead of using the rum-recorder package. Example:
*  `@datadog/browser-rum-recorder/types` (this could work because it is not setup as a real package here)
*  or a relative path `../[...]/packages/rum-recorder/src/types`

Reply:
I'd be in favor of exploring the solution where we don't expose them.